src/components/utilities-texts.js

```diff
@@ -0,0 +1,15 @@
+export function sentResetLinkText() {
+        return "Recovery link has been sent to your email"
+    }
+
+export function changedPasswordText() {
+        return "Password has been changed! \n You will be redirected do login page soon"
+    }
+
+export function expiredTokenText() {
+        return "It looks like your token has expired. \n Reset your password again."
+    }
+
+export function invalidPasswordsText() {
+        return "Provided passwords must be the same and have 8+ chars"
+    }
```

Comment:
Great, all messages in one place. It will be easier to translate it in the future.